doc/get_doc_version.py

```diff
@@ -0,0 +1,15 @@
+import subprocess
+import os
+
+def get_version():
```

Comment:
@icfaust Could you please look into whether this captures the version number correctly.